docs/development/testing.md

```diff
@@ -68,6 +78,12 @@ Use `deactivate` to leave your VE.
 
 ### Configuration
 
+There are two different ways to create a working folder. A manual creation like in the example below or with the bash command:
+
+```bash 
+molecule init role computacenter.webserver-demo --driver-name docker
```

Comment:
The role name does work, executing this command returns an error:
```bash
$ molecule init role computacenter.webserver-demo --driver-name docker
CRITICAL Outside collections you must mention role namespace like: molecule init role 'acme.myrole'. Be sure you use only lowercase characters and underlines. See https://galaxy.ansible.com/docs/contributing/creating_role.html
```
The chapter describes adding the molecule configuration to an existing role, here the command:
```bash
molecule init scenario
```
is more suitable. Note, this command needs to be executed inside the role folder!